docs/FAQ.md

```diff
@@ -101,7 +101,7 @@ payment system.
 
 You might keep account metadata in your existing database (say,
```

Comment:
The change is good from what it was, but why wouldn't we recommend to use your existing database's primary key as the same in TigerBeetle and avoid a mapping?

Reply:
Ah, we actually do - https://github.com/tigerbeetle/tigerbeetle/blob/main/docs/design/data-modeling.md#reuse-foreign-identifier